zilencer/models.py

```diff
@@ -5,15 +5,18 @@
 import zerver.models
 import datetime
 
+API_KEY_LENGTH = 64
```

Comment:
I think it might make sense to add these into the RemoteZulipServer model itself? So e.g. you'd access them as `RemoteZulipServer.API_KEY_LENGTH`.